.github/workflows/next_pull_request_build.yml

```diff
@@ -9,41 +9,10 @@ on:
       - synchronize
 
 jobs:
-
-  test:
-    runs-on: ubuntu-20.04
-
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: node.js 16
-        uses: actions/setup-node@v2
-        with:
-          node-version: 16
-          registry-url: https://registry.npmjs.org/
-          cache: 'npm'
-
-      - name: set up git
-        run: git config --global url."https://github.com/".insteadOf "ssh://git@github.com/"
-
-      - name: installing
-        run: npm ci
-
-      - name: setup environment
-        run: source ./node_modules/.bin/setup-environment $PWD website-dao-dev
-
-      # TODO: enabled linters
-      # - name: linters
-      #   uses: wearerequired/lint-action@v1
-      #   with:
-      #     prettier: true
-      #     prettier_extensions: ts,tsx,js,jsx,json,md
-      #     continue_on_error: false
-
-      - name: testing
-        run: npm test
-        env:
-          DISCOURSE_API_KEY: "DISCOURSE_API_KEY"
-
-      - name: building
-        run: npm run build
+  build:
+    uses: decentraland/actions/.github/workflows/build-quay-main.yml@main
+    with:
+      service-name: governance
+    secrets:
+      QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
+      QUAY_TOKEN: ${{ secrets.QUAY_TOKEN }}
```

Comment:
this whole file shouldn't be necessary anymore I think

Reply:
Removed. It'd be nice to have deployments per pull request but we can review that later after the deploy on merge is working ok as it probably needs a different setup.